Utilities/Log.cpp

```diff
@@ -293,7 +294,6 @@ logs::file_writer::file_writer(const std::string& name)
 		m_fmap = CreateFileMappingW(m_file.get_handle(), 0, PAGE_READWRITE, s_log_size >> 32, s_log_size & 0xffffffff, 0);
 		m_fptr = (uchar*)MapViewOfFile(m_fmap, FILE_MAP_WRITE, 0, 0, 0);
 #else
-		m_file.trunc(s_log_size);
```

Comment:
Don't delete this line, without this line RPCS3 crashes on Linux.

Reply:
Ah must've deleted that by mistake when I was resolving the merge conflict. I've added it back in and will double check I haven't accidentally deleted anything else.